### PR TITLE
Update all dependencies and replace webpack-serve

### PR DESCRIPTION
**Pull request by JordanShurmer**
* Update versions to latest from npm
* Replace deprecated webpack-serve with official webpack-dev-server

### CODE REVIEW

**Comment by YuriiNskyi** [comment]
Did you try running `npm run dev`? It simply shows a list of files, not your desired site. I suppose that it must be `"dev": "webpack-dev-server --content-base public"` command.

File: package.json [hunk 1]
```diff
@@ -3,18 +3,18 @@
...
     "webpack": "^4.8.3",
-    "webpack-cli": "^2.0.14",
-    "webpack-serve": "^1.0.2"
+    "webpack-cli": "^3.1.2",
+    "webpack-dev-server": "^3.1.9"
   },
   "scripts": {
     "build": "cross-env NODE_ENV=production webpack",
-    "dev": "webpack-serve --content public --hot"
+    "dev": "webpack-dev-server"
```
Comment (YuriiNskyi) [inline package.json]:
```suggestion
    "dev": "webpack-dev-server --content-base public"
```
That works for me. Original command simply shows a list of files.

Comment (JordanShurmer) [reply package.json]: Yes. That's correct, thanks!

**Comment by Rich-Harris** [comment]
Thank you!

**Pull request merged**